plugins/msr/fu-plugin-msr.c

```diff
@@ -357,6 +359,8 @@ fu_plugin_add_security_attr_amd_sme_enabled(FuPlugin *plugin, FuSecurityAttrs *a
 
 	if (!priv->amd64_syscfg.fields.sme_is_enabled) {
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENCRYPTED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONFIG_OS);
```

Comment:
I think this and the other SME one are mixed up.  If the MSR doesn't say enabled then it's contact OEM.  If the kernel doesn't say enabled it's turn on in the OS.